src/data/issue_extraction.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.                print('SLEEPING NOW')
</pre><p>Can you add something that indicates how long the sleep will last? like, "SLEEPING NOW RETURN IN AN HOUR"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>